bpf/lib/common.h

```diff
@@ -748,6 +750,25 @@ struct ct_state {
 	__u16 backend_id;	/* Backend ID in lb4_backends */
 };
 
+#define SRC_RANGE_STATIC_PREFIX_V4		\
+	(8 * (sizeof(struct lb4_src_range_key) - sizeof(struct bpf_lpm_trie_key)))
+#define SRC_RANGE_STATIC_PREFIX_V6		\
+	(8 * (sizeof(struct lb6_src_range_key) - sizeof(struct bpf_lpm_trie_key)))
```

Comment:
The ipcache struct is different from `lb{4,6}_src_range_key` in a way that the last field can be used to store either IPv6 or IPv4 addr. So, a macro to calculate the LPM lookup prefix for both would get quite ugly.

Instead of consolidating both, I've made a single `SRC_RANGE_STATIC` to calculate the prefix for `lb{4,6}_src_range_key`.